src/app/Fake.DotNet.Cli/DotNet.fs

```diff
@@ -498,6 +498,8 @@ module DotNet =
             lift (fun o -> { o with Verbosity = verb}) x
         let inline withCustomParams args x =
             lift (fun o -> { o with CustomParams = args}) x
+        let inline withAdditionalArgsAsList args x =
```

Comment:
It's a `seq`, but I'm pretty sure they would get the point.

Reply:
What I meant was for `withX` helpers it is clear they write to `X` for the `withAdditionalArgs` helper it is not really clear what it is doing (well it is clear but doesn't match the scheme). That's why I would either opt for something with `withCustomParams*` prefix or a comment (yes it would stand out but the helper is a bit "special" so that would be fine imho) ;)

Reply:
Done.